client/component/src/DomainConfig.tsx

```diff
@@ -83,11 +85,16 @@ interface DomainConfigProps {
  */
 function DomainConfig(props: DomainConfigProps) {
   const { dataDomain, customDomain, scaleType } = props;
-  const { onCustomDomainChange, histogram } = props;
+  const { onCustomDomainChange, histogramGetterRef } = props;
   const [lockDomain, setLockDomain] = useState<boolean>(false);
 
   const visDomain = useVisDomain(customDomain, dataDomain);
 
+  const histogram = useMemo(
```

Comment:
Does this only update when the ref object changes and not when the function inside the ref changes?